### ISSUE TITLE
cwd-mode plain should truncate from left

### ISSUE THREAD

**Issue by scop** [justjanne/powerline-go#130]
When truncation applies to cwd-mode plain, the path is truncated from the right. In my opinion more interesting parts of the full path are on the right, so it should be truncated from the left instead. This is how fancy already works.

![plain-truncate](https://user-images.githubusercontent.com/109152/57064525-3bded480-6ccf-11e9-91c1-a50d848b2855.png)

So for example, in the above it would be more useful to truncate the dir to `.../some/path` rather than `~/Documents...` (e.g. if cwd is ~/Documents/foo/bar/some/path).

**Comment by justjanne** [comment]
I agree, but with the library we’re currently using for truncating unicode text (as a path may contain e.g. chinese characters or emoji), we can’t truncate from the left

**Comment by garyteh** [comment]
Maybe allow the user to choose which way to truncate?

**Comment by justjanne** [comment]
I agree that it *should* be done, but with our current library, that’s not easily possible. If someone wishes to contribute the code, sure, I’d love to add it though.
